modules/manage/pages/raft-group-reconfiguration.adoc

```diff
@@ -0,0 +1,66 @@
+= Raft Group Reconfiguration in Redpanda
+:description: Learn how the Redpanda Raft group protocol provides consistency and availability during reconfiguration.
+
+Redpanda and Kafka with Raft both use the Raft consensus algorithm to manage cluster metadata. This metadata is stored as messages in an internal topic in a separate Raft cluster instance, called a Raft group. The messages in this topic contain the current set of brokers that are members of a Raft group. In a Raft group, one member is the leader and the others are followers. 
```

Comment:
we name the special metadata raft group a controller. each raft instance both data and metadata forms a raft group.

Reply:
@mmaslankaprv to double check - is that a different `controller` than the one defined here? https://docs.redpanda.com/current/reference/glossary/#controller

Reply:
We have a special partition (partition == raft group i.e. raft + storage) called `redpanda/controller/0` this raft group stores all the metadata that are mentioned. A leader of this Raft group is a controller broker. 
A raft group term is a general term describing single Raft protocol instance